www-apps/airdcpp-webui/airdcpp-webui-2.1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Web interface for airdcpp-webclient"
+HOMEPAGE="https://airdcpp-web.github.io/"
+SRC_URI="https://registry.npmjs.org/${PN}/-/${P}.tgz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+RDEPEND="=net-p2p/airdcpp-webclient-${PV%.*}*"
+
+S="${WORKDIR}/package"
+
+src_install() {
+	insinto "/usr/share/airdcpp/web-resources"
+	doins -r "${S}/dist"/*
```

Comment:
You can use `.` instead of `*` to avoid unnecessary pattern expansion, and just let `doins` catch everything recursively (note: it will also catch dotfiles).